packages/tests/server/react/rsc-prefetch.test.tsx

```diff
@@ -100,13 +101,22 @@ test('rsc prefetch helpers', async () => {
     );
   }
 
-  const utils = render(
+  const utils1 = render(
+    <App>
+      <Parent />
+    </App>,
+  );
+  await waitFor(() => {
+    expect(utils1.container).toHaveTextContent(`__result`);
+  });
+
+  const utils2 = render(
```

Comment:
logs out this from the log I added above:

![image](https://github.com/trpc/trpc/assets/51714798/773468c4-95a0-40b8-bc93-8ae11ee9d8bb)

Reply:
hmmm, this fails even if I revert the proxy caching here